### ISSUE TITLE
[Accessibility]: Git icon before repo name is announced as generic image but is for display purposes only

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35258]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33513

### Problem description

The git icon before the repo name is announced while adding nothing to the navigation experience. 

![image](https://user-images.githubusercontent.com/18282288/167685973-e4039e06-b977-47d3-a5f7-b1ad800742a5.png)

### Expected behavior

Icon should likely be skipped, is purely for decoration

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
Will be closed by https://github.com/sourcegraph/sourcegraph/pull/35617
